packages/config/src/layer2s/loopring.ts

```diff
@@ -76,8 +80,21 @@ export const loopring: Layer2 = {
     stateValidation: RISK_VIEW.STATE_ZKP_SN,
     dataAvailability: RISK_VIEW.DATA_ON_CHAIN,
     upgradeability: RISK_VIEW.UPGRADABLE_YES,
-    sequencerFailure: RISK_VIEW.SEQUENCER_FORCE_EXIT_L1,
-    validatorFailure: RISK_VIEW.VALIDATOR_ESCAPE_MP,
+    sequencerFailure: {
+      ...RISK_VIEW.SEQUENCER_FORCE_EXIT_L1,
+      description:
+        RISK_VIEW.SEQUENCER_FORCE_EXIT_L1.description +
+        ' The sequencer can censor individual deposits, but in such case users can get their funds back.',
+      references: [
+        'https://etherscan.io/address/0x26d8Ba776a067C5928841985bCe342f75BAE7E82#code#L7252',
+      ],
+    },
+    validatorFailure: {
```

Comment:
maybe we want to add delay here as well?

Reply:
done